docs/schema/workflow-element-elements.md

```diff
@@ -41,117 +42,32 @@ The following sections describe attributes, child elements, and parent elements.
+| **InstantiationUrl** | Optional **Text**. Specifies the URL of the initiation form for this workflow. For example:<br><br>`InstantiationURL` = `MyWorkflowInitiationPage.aspx`<br><br>**Note**: SharePoint Foundation supports absolute or server-relative paths in the workflow template definition. All form path URLs must be expressed in one of these formats; for example, an absolute path, such as `"http://site/library/page.aspx"` or a server-relative path, such as `"/layouts/page.aspx"`. SharePoint Foundation does not support link fix-up in workflow template definitions.<br><br>For more information about initiation forms, see [Workflow Association and Initiation Forms (Windows SharePoint Services)](https://msdn.microsoft.com/library/ffdfe5a7-f860-47b0-80e0-2dc2bd36ed38(Office.15).aspx). |
+| **ModificationUrl** | Optional **Text**. Specifies the URL of the form handling the modifications for this workflow. If your workflow includes multiple modifications, you can program the form you specify with this attribute to:<br><br><li> Display a different view of the form, based on the modification identifier passed to this form.<br><li> Redirect to a separate form, based on the modification identifier passed to this form.<br>  </li> <br>For more information about modification forms, see [Workflow Modification Forms (Windows SharePoint Services)](https://msdn.microsoft.com/library/fba5147d-70a7-4f83-8efc-2fe0a5d0b4a7(Office.15).aspx). |
+| **StatusUrl** | Obsolete. Do not use. |
+| **TaskListContentTypeId** | Optional **Text**. Specifies the content type ID of the content type assigned to the workflow task list.<br><br>For more information about task forms, see [Workflow Task Forms (Windows SharePoint Services)](https://msdn.microsoft.com/library/327137dd-9a2b-4df6-be48-582a4b41aaa9(Office.15).aspx). |
 
 ### Child elements
 
-<table>
-<colgroup>
-<col width="100%" />
-</colgroup>
-<tbody>
-<tr class="odd">
-<td align="left"><p><a href="associationdata-element-workflow.md">AssociationData Element (Workflow)</a></p></td>
-</tr>
-<tr class="even">
-<td align="left"><p><a href="metadata-element-workflow.md">MetaData Element (Workflow)</a></p></td>
-</tr>
-<tr class="odd">
-<td align="left"><p><a href="categories-element-workflow.md">Categories Element (Workflow)</a></p></td>
-</tr>
-</tbody>
-</table>
+- [AssociationData Element (Workflow)](associationdata-element-workflow.md)
```

Comment:
According to the repo owner, he wants tables to remain tables, and lists to remain lists.  Use "Elements" as the single column header.